components/MainNavBar.vue

```diff
@@ -80,6 +80,7 @@
            { name : 'References', url : '/reference', external: false },
```

Comment:
Getting the same nuxt kit issue for this as I was for the previous PR. I also note that the `npm run dev` doesn't work because the script it runs is `nuxt` instead of `nuxt dev`. Could you look at this please? Thanks Ruby